sql/analyzer/analyzer.go

```diff
@@ -338,21 +338,42 @@ func (a *Analyzer) PopDebugContext() {
 // Analyze applies the transformation rules to the node given. In the case of an error, the last successfully
 // transformed node is returned along with the error.
 func (a *Analyzer) Analyze(ctx *sql.Context, n sql.Node, scope *Scope) (sql.Node, error) {
+	return a.analyzeWithSelector(ctx, n, scope, func(string) bool { return true })
```

Comment:
Probably extract a top-level function for this anon function, "analyzeAll" or similar